build.sbt

```diff
@@ -10,6 +14,17 @@ lazy val root = tlCrossRootProject.aggregate(crystal)
 lazy val crystal = crossProject(JVMPlatform, JSPlatform)
   .in(file("."))
   .settings(
+    mimaBinaryIssueFilters ++= Seq(
```

Comment:
Why is this necessary?

Reply:
It seems to me it might be better to just bump the baseVersion.

Reply:
Also fine by me. I added it because the `Semigroup` and `Invariant` were two separate given values, now combined. But mima sees them as "removed". I can either remove this line after the next release (I think) or bump the baseVersion